substrate/frame/bounties/src/lib.rs

```diff
@@ -986,8 +1024,7 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 		let index = BountyCount::<T, I>::get();
 
 		// reserve deposit for new bounty
-		let bond = T::BountyDepositBase::get() +
-			T::DataDepositPerByte::get() * (bounded_description.len() as u32).into();
+		let bond = Self::calculate_bounty_deposit(&bounded_description);
```

Comment:
FYI I started to expose such logics as `#[view_functions]`, not sure if @re-gius and @jsdw sort of agree that deposits are no-brainer users of view-functions, but IMO they are :) especially if the logic is not trivial. As in, if the JS/Rust application needs to mimic some calculation, which in this case it does.